Rakefile

```diff
@@ -28,12 +28,14 @@ RuboCop::RakeTask.new
 
 desc "Pull the latest versions of all dependencies into the gem for distribution"
 task :vendor_deps do
-  TEST_TRACK_JS_CLIENT_VERSION = '1.3.8'.freeze
+  TEST_TRACK_JS_CLIENT_VERSION = '2.0.0-alpha.2'.freeze
```

Comment:
This will need to be updated once this (https://github.com/Betterment/test_track_js_client/pull/23) lands and we cut a `2.0.0-alpha.3` release.